clusterloader2/pkg/tuningset/global_qps_load.go

```diff
@@ -59,7 +59,10 @@ func newGlobalQPSLoad(params *api.GlobalQPSLoad) *globalQPSLoad {
 func (ql *globalQPSLoad) Execute(actions []func()) {
 	var wg wait.Group
 	for i := range actions {
-		ql.limiter.Wait(context.TODO())
+		err := ql.limiter.Wait(context.TODO())
+		if err != nil {
+			return
```

Comment:
This is silently ignoring the error, and skipping all other actions - that's definitely not right.

```
klog.Errorf("Error while waiting for rate limitter: %v - skipping the action", err)
continue
```